app.py

```diff
@@ -27,19 +50,38 @@ def get(self, restaurant_id):
         for restaurant in restaurants:
             if restaurant["id"] == restaurant_id:
                 return jsonify(restaurant), 200
+        raise(HandledError(status_code=404, description="Page doesn't exist", name="Not Found"))
```

Comment:
Error description should be descriptive, in this case it is like "Restaurant with such ID does not exist"

Reply:
And why did you add extra parentheses around?
`raise(HandledError(status_code=404, description="Page doesn't exist", name="Not Found"))`
and did not add a space between `raise` and exception

Reply:
fixed